config/environments/development.rb

```diff
@@ -12,7 +12,7 @@
   config.eager_load = false
 
   # Show full error reports.
-  config.consider_all_requests_local = true
+  config.consider_all_requests_local = false
```

Comment:
🙊

Reply:
Why are we changing this? Surely we want this to be true for development so we can see more detailed error logs for debugging? 
https://guides.rubyonrails.org/configuring.html#config-consider-all-requests-local